### PR TITLE
Make text content of static pages editable.

### PR DESCRIPTION
**Pull request by sekoskel**
Use parler to store and retrieve multilingual content for static pages making them editable via admin.
Added ckeditor wysiwyg to ease text markup.

### CODE REVIEW

**Comment by juyrjola** [comment]
It's better to include ckeditor with `django-npm` or `django-bower`. Keeping a copy in the kuvaselaamo repo is wasteful.

**Comment by sekoskel** [comment]
ckeditor is now added via bower.

**Comment by sekoskel** [comment]
So i've added ckeditor via bower. Anything else i should fix? @juyrjola

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,4 @@ mediaroot/*
 node_modules
 parts
 staticroot/*
+bower_components
```
Comment (juyrjola) [inline .gitignore]: Since we want to match it only in the project root, it should be: `/bower_components/`

File: hkm/static/hkm/vendor [hunk 2]
```diff
@@ -0,0 +1 @@
+../../../bower_components/
```
Comment (juyrjola) [inline hkm/static/hkm/vendor]: It's better to use django-bower's static files finder for this. See django-bower's installation docs on how to enable it.

**Comment by sekoskel** [comment]
Ok, i've replaced direct npm bower install with django-bower wrapper thingy.

**Pull request merged**